tests/NLog.UnitTests/LayoutRenderers/CallSiteTests.cs

```diff
@@ -1040,6 +1040,72 @@ public void CallSiteShouldWorkEvenInlined()
             Assert.Equal("NLog.UnitTests.LayoutRenderers.CallSiteTests.CallSiteShouldWorkEvenInlined", callSite);
         }
 
+#if ASYNC_SUPPORTED
+        [Fact]
+        public void LogAfterAwait_CleanNamesOfAsyncContinuationsIsTrue_ShouldCleanMethodName()
+        {
+            LogManager.Configuration = CreateConfigurationFromString(@"
+                <nlog>
+                    <targets><target name='debug' type='Debug' layout='${callsite:classname=false:cleannamesofasynccontinuations=true}' /></targets>
+                    <rules>
+                        <logger name='*' levels='Debug' writeTo='debug' />
+                    </rules>
+                </nlog>");
+
+            AsyncMethod5().GetAwaiter().GetResult();
+
+            AssertDebugLastMessage("debug", nameof(AsyncMethod5));
```

Comment:
I prefer using fixed strings (in tests) and no `nameof`, `typeof` etc. Could you please change that? (Also below)

Reply:
Sure can do